src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/Vault/SetAzureRmRecoveryServicesAsrVaultContext.cs

```diff
@@ -76,16 +72,24 @@ public override void ExecuteSiteRecoveryCmdlet()
             {
                 using (var powerShell = System.Management.Automation.PowerShell.Create())
```

Comment:
@viverm this `powershell` object isn't used anymore, so we should get rid of the `using` that instantiates it

Reply:
File name changed from NewAzureRMRecoveryServicesVault.cs to NewAzureRmRecoveryServicesVault.cs reindex ed it.